### DISCUSSION TITLE
CLIP based "Images like this"

### DISCUSSION THREAD

**Discussion by lambadalambda** [ente-io/ente#633] [Enhancements]
I often want to find images that are similar to a given images, given that CLIP embeddings are already generated for the semantic search, it would be nice to have something like '10 images similar to this one'.

**Comment by eric-gitta-moore** [comment]
+1. But perhaps limited by the capabilities of the model? Or a corresponding retrieval service is required. I understand that a service similar to elastic search might be needed to store the results of apple ml mobileclip
